Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -350,19 +421,43 @@ void MaxEnt::exec() {
   iterationCounts.reserve(nSpec);
   outEvolChi->setPoints(0, Points(nIter, LinearGenerator(0.0, 1.0)));
 
+  size_t dataLength = complexData ? 2 * inWS->y(0).size() : inWS->y(0).size();
+
   for (size_t spec = 0; spec < nSpec; spec++) {
 
     // Start distribution (flat background)
     std::vector<double> image(npoints, background);
 
-    std::vector<double> data;
-    std::vector<double> errors;
+    std::vector<double> data(dataLength, 0.0);
+    std::vector<double> errors(dataLength, 0.0);
     if (complexData) {
-      data = toComplex(inWS, spec, false);  // false -> data
-      errors = toComplex(inWS, spec, true); // true -> errors
+      data = toComplex(inWS, spec, false, sumSpectra); // 3rd arg false -> data
+      errors =
+          toComplex(inWS, spec, true, sumSpectra); // 3rd arg true -> errors
     } else {
-      data = inWS->y(spec).rawData();
-      errors = inWS->e(spec).rawData();
+      if (sumSpectra) {
+        const size_t numBins = inWS->y(0).size();
+        data.reserve(numBins);
+        errors.reserve(numBins);
+        for (size_t s = 0; s < nSpecSum; s++) {
+          for (size_t i = 0; i < numBins; i++) {
+            data[i] += inWS->y(s).rawData()[i];
+            errors[i] += inWS->e(s).rawData()[i];
+          }
+        }
+      } else {
+        data = inWS->y(spec).rawData();
+        errors = inWS->e(spec).rawData();
+      }
+    }
```

Comment:
am I right in thinking that this means if you sum the spectra then the chi^2 is a comparison between the summed real data and the summed calculated results (time domain)?

Reply:
Yes. I've taken to be effectively a one spectrum comparison on the sum of all the spectra.